Neos.Cache/Classes/Backend/SimpleFileBackend.php

```diff
@@ -528,7 +540,7 @@ protected function writeCacheFile($cacheEntryPathAndFilename, $data)
                 clearstatcache(true, $cacheEntryPathAndFilename);
                 return $result;
             }
-            usleep(rand(10, 500));
+            usleep(random_int(10, 500));
```

Comment:
IMHO this is not a good idea because random_int is slower than rand and no cryptographically secure value is needed here.